SocialCareCaseViewerApi.Tests/V1/Controllers/CaseControllerTests.cs

```diff
@@ -83,5 +92,64 @@ public void ListCasesReturns404WhenNoCasesAreFound()
             response?.StatusCode.Should().Be(404);
             response?.Value.Should().Be("Document Not Found");
         }
+
+        [Test]
+        public void GetCaseByRecordIdDoesNotCallTheCreateRequestAuditUseCaseWhenAuditingIsEnabledIsFalse()
+        {
+            var request = new GetCaseNotesRequest();
```

Comment:
```suggestion
            var request = new GetCaseNotesRequest()
             {
                 AuditingEnabled = false,
                 UserId = _faker.Person.Email,
                 Id = "tyut67t89t876t",
                 ResidentId = "4"
             };
```

I think this is more reflective of the test description we have. It also might be worth adding a test which just has `Id` to prove we're backwards compatible.